modules/tinymce/src/core/main/ts/DragDropOverrides.ts

```diff
@@ -187,17 +191,73 @@ const moveGhost = (
         });
         // This basically means that the mouse is close to the left edge
         // (within MouseRange pixels of the left edge)
-      } else if (mouseX - MouseRangeToTriggerScroll <= 0) {
+      } else if (mouseX - mouseRangeToTriggerScrollInsideEditor <= 0) {
         const scrollLeft = (currentLeft: number) => {
           win.scroll({
-            left: currentLeft - ScrollPixelsPerInterval,
+            left: currentLeft - scrollPixelsPerInterval,
             behavior: 'smooth'
           });
         };
         state.intervalId.set(() => {
           const currentLeft = win.scrollX;
           scrollLeft(currentLeft);
         });
+        // This basically means that the mouse is close to the bottom edge
+        // of the page (within MouseRange pixels) when the bottom of
+        // the editor is offscreen
+      } else if (outerMouseY + mouseRangeToTriggerScrollOutsideEditor >= window.innerHeight) {
+        const scrollDown = (currentTop: number) => {
+          window.scroll({
```

Comment:
Also I first thought this would be a to simplistic view of scrolling things since in theory you could have any number of parent scrollable containers. However did some experimentation and it seems that native drag scrolling only scrolls the closest view at least on Chrome. So if the point is outside of the parent viewport then that is scrolled if it's outside the inner viewport then that is scrolled. But that is kind of what is implemented just find it a bit strange that you can drag scroll and end up with no scroll at all even if there are scrollbars. But since we try to mimic the native behavior here I think this should be fine.